### ISSUE TITLE
ScrollViewer and any other scroll items view not scroll using mouse wheel

### ISSUE THREAD

**Issue by REDECODE** [microsoft/WinUI-Gallery#1064]
### Which version of the app?

WinUI 3 Gallery

### Description

Rotating the mouse wheel when i have the mouse hover on a list with scrollbar does nothing, but i can scroll sliding the scrollbar pressing and sliding on the scrollbar. On other apps (not from the store) the mouse wheel works perfectly. I can reproduce this bug creating an empty MAUI App with a scrollview.

### Windows version

May 2021 Update/21H1 (19043)

### Additional context

Using the WinUI 2 Gallery mouse wheel works as expected. Issue only on WinUI 3

**Comment by Jay-o-Way** [comment]
Are you on Win 10 or Win 11?

**Comment by bpulliam** [comment]
In the linked PowerToys bug, it mentions:
Found the problem, having X-Mouse Button Control software running even if you didn't remap the wheel up and wheel down key it blocks it from scroll for some reason. (this does not happen anywhere else like chrome and other softwares).

Are you running XMouseButtonControl?

**Comment by REDECODE** [comment]
Hello, thanks for the reply!
Yes im'm using PowerToys but quite everything (including mouse utility) is disabled, i have enabled only color picker, text extractor and image resize. I even try to close it but it doesnt work. Where i can find XMouseButtonControl ?

**Comment by bpulliam** [comment]
XMouseButtonControl is a utility, separate from Powertoys or WinUI Gallery.  If you don't have that installed, then something else is going on. We will be releasing a new store update to coincide with the release of WinAppSDK 1.2 very soon. If you don't have that utility, let's see if the issue persists after the store update.

**Comment by techIndia-hacker** [comment]
@REDECODE Yes the scroll using mouse doesn't work at all. I have synaptics mouse touchpad. No software except synaptics driver is installed regarding mouse

**Comment by karkarl** [comment]
@REDECODE @techIndia-hacker Does the bug repro in WinUI galler, and if so, what page and sample? If not, can you attach your sample app.

**Comment by techIndia-hacker** [comment]
Pls provide steps to what exactly i need to so to helo you

**Comment by REDECODE** [comment]
Well.. don't know if it's a framework update, Windows updates or WinUI Gallery updates but now the scroll workrs perfectly using the same pc and same mouse of 1 years ago. 
Steps for reproduce was open WinUI gallery, shrink the window app untill you see the lateral scroll on the menu on the left and try to scroll with the wheel of the mouse, but now it works perfectly for me. 
I can close this issue

**Comment by REDECODE** [comment]
can't see any app called Files, unless you refer to "Files App" but it's not free to download

**Comment by AndrewKeepCoding** [comment]
We can close this issue. It's mentioned by @REDECODE that it's not reproducible anymore. I've also confirmed it on v2.6.
